github-check-releases/README.md

```diff
@@ -0,0 +1,117 @@
+# GitHub Check Releases
+
+This directory contains tooling to enable us to check GitHub releases programmatically, and validate them against the
+version(s) of the product(s) we use in our code example test suite(s).
+
+This is a simple PoC with three main components:
+
+- A [JSON file](repo-details.json) that contains the list of products, GitHub repo information, and test suite versions.
+- It uses [octokit](https://github.com/octokit/octokit.js) to get release data for those MongoDB products from GitHub, and compare it against the version 
+  in our JSON file.
+- It uses [axios](https://github.com/axios/axios) with the Jira API to automatically create tickets to update test suite 
+  versions.
+
+## Add or change products and test suite versions
+
+This project pulls data from [repo-details.json](repo-details.json) to check the release versions for the MongoDB
+products we use in our code example test suites. It deserializes the data using `RepoDetails` declared in the
+[RepoDetails file](RepoDetails.js) to map the owner and repo name for a given product whose version we want to check.
+
+### Add a new product
+
+To add a new product, create a new entry for it in the `repo-details.json` file. 
```

Comment:
```suggestion
To add a new product, create a new entry in the `repo-details.json` file in the following format:

`
    "owner": "<repo-owner>",
    "repo": "<repo-name>",
    "productName": "<MongoDB Product Name>",
    "testSuiteVersion": "<x.y.z>"
`
  
```
(this prob won't format correctly because of the suggestion syntax, but you get the idea)